es_ES/LC_MESSAGES/mail_new_friend.po

```diff
@@ -4,17 +4,17 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "NEW_FRIEND"
-msgstr "New Flipnote Friend!"
+msgstr "¡Nuevo amigo de Flipnote!"
 
 msgid "NEW_FRIEND_INFO"
-msgstr "One of your Flipnote Friends, %s, is on Sudomemo! How cool is that?"
+msgstr "¡Uno de tus amigos de Flipnote, %s, está en Sudomemo! ¿No es genial?"
```

Comment:
I don't think I've heard "No es genial" used like that before, we might want to stick to something like "Que genial" instead.